tests/opentelemetry-tests/README.rst

```diff
@@ -0,0 +1,10 @@
+OpenTelemetry Test Utilities
+============================
+
+This package provides internal testing utilities for the OpenTelemetry Python project and provides no stability or quality guarantees.
```

Comment:
We should consider not using semantic versioning for packages that do not provide a public API, because providing one is a [requirement](https://semver.org/#spec-item-1) of semantic versioning.

Reply:
Do you mean we should declare in docs that this package does not follow semver? The package does provide a public API though. That's what otel tests use, right? Besides following semver is useful as we can keep the package on 0.x forever and it'll be obvious that it is unstable. If we don't follow semver we'll have to communicate that the package is unstable in some other way. 

Either way not a big issue for me. As long as the unstable nature of the package is obvious to people when they find the package on pypi, I'm happy with any versioning scheme.

Reply:
Ah, yes, we could keep it in 0.X.Y. :v: